README.md

```diff
@@ -173,6 +173,9 @@ jobs:
 
       # Optional, sends a slack notification to the channel specified in the repository secrets
       slack-notification: true
+
+      # Optional, the path to the Dockerfile you wish to build. Defaults to Dockerfile at the repository root.
+      docker-dockerfile-path: "./path/to/Dockerfile"
```

Comment:
Please do not modify the README manually, it happens automatically if you follow the [contribution page](https://github.com/dfds/shared-workflows/blob/master/docs/CONTRIBUTING.md)

Basically you need to update the example file :)

Reply:
That's cool. I've reverted the commit for the README and updated the example. I've also made some additions to the contributing guide to hopefully make it clearer